packages/twenty-front/src/modules/object-metadata/hooks/useFieldMetadataItem.ts

```diff
@@ -45,6 +45,7 @@ export const useFieldMetadataItem = () => {
       'id' | 'label' | 'icon' | 'description' | 'defaultValue'
     > & {
       options?: FieldMetadataOption[];
+      type: FieldMetadataType;
```

Comment:
what else is left in Fieldmetadata? should we go with an Omit<> instead? I prefer Pick as I find it more explicit but if it's almost pick everything it might not worth it

Reply:
There is 11 parameters left, I will leave the Pick but move the type and options in the Pick when possible